packages/bindings/npm/linux-arm64-gnu/package.json

```diff
@@ -28,6 +28,6 @@
     "provenance": true
   },
   "engines": {
-    "node": ">=16"
+    "node": ">=16.13.0"
```

Comment:
Node 16.13 is the first Node 16 LTS and it properly supports ESM. Every previous 16.x release has been unsupported for years.